src/Workspaces/Core/Desktop/Workspace/Storage/StorageOptions.cs

```diff
@@ -14,6 +9,12 @@ internal static class StorageOptions
         public const string OptionName = "FeatureManager/Storage";
 
         public static readonly Option<StorageDatabase> Database = new Option<StorageDatabase>(
-            OptionName, nameof(Database), defaultValue: StorageDatabase.Esent);
+            OptionName, nameof(Database), defaultValue: StorageDatabase.SQLite);
+
+        /// <summary>
+        /// Threshold to start to use a DB (50MB)
+        /// </summary>
+        public static readonly Option<int> SolutionSizeThreshold = new Option<int>(
+            OptionName, nameof(SolutionSizeThreshold), defaultValue: -1);
```

Comment:
:question: Why is the default value -1? Small projects with a **.gitignore** containing `*.suo` currently don't need to exclude `.vs/`, which means many likely don't ignore this specifically. Changing the default value so the on-disk database is always created would cause this file to show up as changed for all these users.